source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -90,6 +91,8 @@ public void create() {
         spawnMinotaur();
         spawnGhostKing();
 
+        playMusic();
```

Comment:
Remove music

Reply:
We added muting and music volume controls, that's why we added it back in